### PR TITLE
add blog palfish-vldb-paper

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### CODE REVIEW

**Comment by ran-huang** [comment]
@CaitinChen PTAL. Thanks~

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: 'Making an HTAP Database a Reality: What I learned from PingCAP’s VLDB paper'
```
Comment (CaitinChen) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:2]:
```suggestion
title: 'Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper'
```

Comment (CaitinChen) [reply making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:2]: Please update all quotes to straight quotes.

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+date: 2020-10-22
+summary: 'Recently, VLDB published the PingCAP paper, TiDB: A Raft-based HTAP Database. This article, a DBA at PalFish shares his thoughts on the article and his expectations for TiDB’s future development.'
+tags: ['HTAP', 'VLDB', 'Consistency', 'Raft']
+categories: ['HTAP']
+image: /images/blog/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg
+---
+
+**Author:** Xianlin Chen (DBA at PalFish)
+
+![banner](media/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg)
```
Comment (CaitinChen) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:13]:
```suggestion
![HTAP database](media/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg)
```

Comment (CaitinChen) [reply making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:13]: Next time, you can use a shorter image name with SEO keywords.

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Recently, [VLDB 2020](https://vldb2020.org/) published [PingCAP](https://pingcap.com/)’s paper, [TiDB: A Raft-based HTAP Database](https://en.pingcap.com/blog/vldb-2020-tidb-a-raft-based-htap-database). This is the first paper in the industry to describe the implementation of a distributed [Hybrid Transactional/Analytical Processing](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing) (HTAP) database. As a DBA who benefits greatly from [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source, distributed SQL database, I’m happy that VLDB recognized TiDB, and I’m inspired by the PingCAP engineering team’s novel ideas.
```
Comment (CaitinChen) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:15]:
```suggestion
Recently, [VLDB 2020](https://vldb2020.org/) published [PingCAP](https://pingcap.com/)’s paper, [TiDB: A Raft-based HTAP Database](https://pingcap.com/blog/vldb-2020-tidb-a-raft-based-htap-database). This is the first paper in the industry to describe the implementation of a distributed [Hybrid Transactional/Analytical Processing](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing) (HTAP) database. As a DBA who benefits greatly from [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source, distributed SQL database, I’m happy that VLDB recognized TiDB, and I’m inspired by the PingCAP engineering team’s novel ideas.
```

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md (hunk 2 above)
Comment (CaitinChen) [reply making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:13]: Descriptive ALT text with an SEO keyword helps our SEO performance, so I want to use "HTAP database."

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+| **Characteristics** | **OLTP** | **OLAP** |
+|:--|--|--|
+| Data queried in one request | Small (a few rows) | Large | 
+| Real-time update | Yes | No |
+| Transactions | Yes | No | 
+| Concurrency | High | Low |
+| Query pattern | Similar | Varies a lot |
+
+<div class="caption-center">Comparing OLTP and OLAP workloads</div>
```
Comment (CaitinChen) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:35]: Generally, we don't add table captions in our posts.

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:35]:
```suggestion
```

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+tags: ['HTAP', 'VLDB', 'Consistency', 'Raft']
+categories: ['HTAP']
+image: /images/blog/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg
+---
+
+**Author:** Xianlin Chen (DBA at PalFish)
+
+![HTAP database](media/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg)
+
+Recently, [VLDB 2020](https://vldb2020.org/) published [PingCAP](https://pingcap.com/)'s paper, [TiDB: A Raft-based HTAP Database](https://en.pingcap.com/blog/vldb-2020-tidb-a-raft-based-htap-database). This is the first paper in the industry to describe the implementation of a distributed [Hybrid Transactional/Analytical Processing](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing) (HTAP) database. As a DBA who benefits greatly from [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source, distributed SQL database, I'm happy that VLDB recognized TiDB, and I'm inspired by the PingCAP engineering team's novel ideas.
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:15]:
```suggestion
Recently, [VLDB 2020](https://vldb2020.org/) published [PingCAP](https://pingcap.com/)'s paper, [TiDB: A Raft-based HTAP Database](https://pingcap.com/blog/vldb-2020-tidb-a-raft-based-htap-database). This is the first paper in the industry to describe the implementation of a distributed [Hybrid Transactional/Analytical Processing](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing) (HTAP) database. As a DBA who benefits greatly from [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source, distributed SQL database, I'm happy that VLDB recognized TiDB, and I'm inspired by the PingCAP engineering team's novel ideas.
```

**Comment by ran-huang** [comment]
@CaitinChen Thanks! All comments addressed.

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
+---
+title: 'Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper'
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:2]:
```suggestion
title: Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper
```

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
+---
+title: Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:2]:
```suggestion
title: "Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper"
```

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
+---
+title: "Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper"
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:2]:
```suggestion
title: [Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper]
```

File: making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
+---
+title: [Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper]
+author: ['Xianlin Chen']
+date: 2020-10-22
+summary: 'Recently, VLDB published the PingCAP paper, TiDB: A Raft-based HTAP Database. This article, a DBA at PalFish shares his thoughts on the article and his expectations for TiDB's future development.'
+tags: ['HTAP', 'VLDB', 'Consistency', 'Raft']
+categories: ['HTAP']
+image: /images/blog/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg
+---
```
Comment (ran-huang) [inline making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md:9]:
```suggestion
title: Making an HTAP Database a Reality: What I Learned from PingCAP's VLDB Paper
author: ['Xianlin Chen']
date: 2020-10-22
summary: Recently, VLDB published the PingCAP paper, TiDB: A Raft-based HTAP Database. This article, a DBA at PalFish shares his thoughts on the article and his expectations for TiDB's future development.
tags: ['HTAP', 'VLDB', 'Consistency', 'Raft']
categories: ['HTAP']
image: /images/blog/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg
---
```

**Pull request merged**